mimic-iii-carevue/install.sh

```diff
@@ -0,0 +1,578 @@
+        psql -U postgres -c "create user $(whoami) with superuser createdb createrole replication bypassrls password '$(whoami)';"
+        echo ""
+
+        echo "--- The database user $(whoami) has been created successfully. ---"
+        echo "--- The default password of the database user $(whoami) has been set as:：$(whoami) ---"
+        echo ""
+    fi
+
+
+    # create database
+    if psql -U postgres -t -c '\l' | cut -d \| -f 1 | grep -qw "$(whoami)"; then
+        # The database with the same name as the system user exists
+        echo ""
+    else
+        # create database with the same name
+        psql -d postgres -c "create database $(whoami) owner $(whoami);"
+        echo ""
+        
+        echo "--- The database with the same name as the system user created successfully. ---"
+        echo "--- You just type 'psql' in the terminal to login Postgresql without any other arguements. ---"
+        echo ""
+    fi
+
+else
+    # Chinese
+
+    # create database user
+    if psql -U postgres -t -c '\du' | cut -d \| -f 1 | grep -qw "$(whoami)"; then
+        # the database user with the same name as the system user exists
+        echo ""
+    else
+        # the database user with the same name as the system user does not exist
+        echo "--- 与系统同名的数据库用户不存在，准备以系统用户为名创建数据库用户 ---"
+        echo ""
+
+        # create database user with the same name as the system user
+        psql -U postgres -c "create user $(whoami) with superuser createdb createrole replication bypassrls password '$(whoami)';"
+        echo ""
+
+        echo "--- 成功创建与系统用户同名的数据库用户$(whoami)，用户$(whoami)默认密码已经被设置为：$(whoami) ---"
+        echo ""
+    fi
+
+
+    # create database
+    if psql -U postgres -t -c '\l' | cut -d \| -f 1 | grep -qw "$(whoami)"; then
+        # The database with the same name as the system user exists
+        echo ""
+    else
+        # create database with the same name
+        psql -d postgres -c "create database $(whoami) owner $(whoami);"
+        echo ""
+        
+        echo "--- 成功创建与系统用户同名的数据库(方便使用终端登录) ---"
+        echo "--- 你只需在终端输入“psql”即可登录，无需其他任何参数 ---"
+        echo ""
+    fi
+    
+fi
+
+
+
+
+# install the database
+if [[ $lan_sel == 1 ]]; then
+    # English
+
+    echo -e "===================================="
+    echo -e 'Any message like "NOTICE: TABLE "XXXXXX" does not exist" during the installation is normal, do not worry about it!'
+    echo -e "===================================="
+    echo ""
+
+    # function of timing
+    secs_to_human() {
+        echo "It takes $(( ${1} / 3600 ))h $(( (${1} / 60) % 60 ))m $(( ${1} % 60 ))s to install the $obj"
+    }
+
+    echo "--- Install basic tables of database ---"
+    echo ""
+    export OP='dbname=mimic3_carevue options=--search_path=mimiciii_cv,mimiciii_derived'
+
+    # build_starttime
+    build_starttime=$(date +%s)
+
+    # drop database if exists
+    psql -c "drop database if exists mimic3_carevue;"
```

Comment:
The database name, schema name, and derived schema name should be environment variables set at the top of the script. This will allow you to add a --carevue flag which changes these variables to carevue specific ones.

I would suggest the database remains simple (`mimic`), but the schema is updated to `mimiciii_cv`, and the derived schema is also changed `mimiciii_cv_derived`.